File: FindWeather/Location.java
class Location {
    private final double latitude;
    private final double longitude;
    private final String city;

    public Location(double latitude, double longitude, String city) {
        this.latitude = latitude;
        this.longitude = longitude;
        this.city = city;
    }

    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public String getCity() {
        return city;
    }
}

File: FindWeather/Lab-8_FindWeather/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab_8_FindWeather
{
    public class Location
    {
        public string City { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }

        public Location(string city, double latitude, double longitude)
        {
            City = city;
            Latitude = latitude;
            Longitude = longitude;
        }

        public string GetCity()
        {
            return City;
        }


        public double GetLatitude()
        {
            return Latitude;
        }

        public double GetLongitude()
        {
            return Longitude;
        }

        public void ShowInfo()
        {
            Console.WriteLine($"City: {City}, Latitude: {Latitude}, Longitude: {Longitude}");
        }

    }
}
